fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/literal/Literal.java

```diff
@@ -82,6 +82,10 @@ public static Literal of(Object value) {
 
     public abstract Object getValue();
 
+    public double getDouble() {
```

Comment:
add java doc to this function to explain what it is use for

Reply:
done